wrapper/test/test1.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
+const { expect } = require("chai");
+const { execSync } = require("child_process");
+
+/*
```

Comment:
@ddelpiano can you scrub these please?

Reply:
hey @tgbugs , the above are currently used for the test.
I can clean them but in that case we can remove the test1.js for good since that won't work anymore without them, let me know what you prefer.

Reply:
Could they be passed as an environment variable?

Reply:
not really, the file itself is the entrypoint for mocha when the tests are ran.
As said, we can remove the file for good or comment this entirely and then when you prefer/want you can restore it (if ever needed).

Reply:
Hrm, maybe we are talking about opposite things, I'm referring to the sheet ids below.

Reply:
Hey Tom, apologies for the delay, I agree, I was talking about the dependencies on top :-)
I will change the script later today / tomorrow morning to get the ids as parameter and update this PR.

Thanks!